packages/gestalt/src/ErrorFlyout/__tests__/__snapshots__/ErrorFlyout.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ErrorFlyout renders 1`] = `null`;
```

Comment:
Seems like this test isn't doing what we want since its returning `null` which doesnt prove it rendered? Should this set of props be a test for `ErrorFlyout does not render`? Then we actually need to write another test for `ErrorFlyout renders`

Reply:
@pfarejowicz Thanks for catching that - should be fixed now.